### ISSUE TITLE
How do I connect to a Bluetooth cube on the web version of TPS cube?

### ISSUE THREAD

**Issue by jhale1805** [D0ntPanic/tpscube#12]
I'm seeing claims on this web page that tpscube supports connections to Bluetooth cubes, but I'm not seeing an option to connect to any such cube when I visit the main web page at https://tpscube.xyz/

I am seeing an old commit suggesting that this feature is only available in the native version of the application. Is that still true? If so, what blockers are stopping this from being a reality in the web app, OR is it possible to use a GitHub Actions pipeline to create releases of the native applications so that potential users such as myself don't have to install Rust and build the app to be able to start using it?

I have zero experience with Rust, but I am a sub 15 solver working on several other cubing projects myself and happy to help with permissively licensed projects like this one as best I can.

**Comment by galenbwill** [comment]
Yes, it's still true that this is only available in native builds. iirc it's because of limitations in the BTLE stack being used.

But I agree it would be really nice to have it fixed.

(Also, nice on the sub-15!)
